src/main/java/jenkins/plugins/slack/SlackNotifier.java

```diff
@@ -25,6 +24,7 @@
 import java.util.logging.Level;
 import java.util.logging.Logger;
 import java.util.stream.Stream;
+import javax.annotation.CheckForNull;
```

Comment:
please use `edu.umd.cs.findbugs.annotations.CheckForNull`, the javax ones have a licensing issue.

Reply:
I followed the comment from the class which refers that the annotation is deprecated:
```
/**
 * The annotated element might be null, and uses of the element should check for
 * null.
 *
 * When this annotation is applied to a method it applies to the method return
 * value.
 *
 * @deprecated - use {@link javax.annotation.CheckForNull} instead.
 **/
@Documented
@Target({ ElementType.FIELD, ElementType.METHOD, ElementType.PARAMETER, ElementType.LOCAL_VARIABLE })
@Retention(RetentionPolicy.CLASS)
@javax.annotation.Nonnull(when = When.MAYBE)
@TypeQualifierNickname
@Deprecated
public @interface CheckForNull {

}
```